build.gradle

```diff
@@ -41,6 +41,9 @@ allprojects {
         // Use Scala 2.12 in our library project
         implementation 'org.scala-lang:scala-library:2.12.7'
         implementation "com.typesafe.scala-logging:scala-logging_$scalaVersion:3.9.2"
+        implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
```

Comment:
I'm not really fond of prescribing a logging backend in a library. I feel we should stick to slf4j as much as possible.

Reply:
yeah I am also not 100% sure about that at the same time I don't feel strongly about having this here. Is it possible to override it as a user of a library? At least by providing a different logback config?

But I agree that libraries should be as lightweight in terms of dependencies as possible :)